week1.js3/index.js

```diff
@@ -0,0 +1,43 @@
+async function fetchData() {
+    try {
+        const response = await fetch("https://raw.githubusercontent.com/najmeh25/najmeh25.github.io/refs/heads/main/index.json");
+        return await response.json();
+    } catch {
+        console.error("Failed to fetch data.");
+        return [];
+    }
+}
+
+// Display data in the list
+function displayData(data) {
+    const dataList = document.getElementById("data-list");
+    dataList.innerHTML = data.length
+        ? data.map(item => `
+            <li>
```

Comment:
Remember that you need to create HTML elements by using 
https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement and https://stackoverflow.com/questions/11351135/create-ul-and-li-elements-in-javascript